### PR TITLE
Mention required baseUrl everywhere

### PR DESCRIPTION
**Pull request by jkoenig134**

### CODE REVIEW

File: _docs_operate/setup-with-helm-charts.md [hunk 1]
```diff
@@ -130,6 +132,11 @@ config:
 pod:
   connector:
     environment:
+      - name: transportLibrary__baseUrl
+        alueFrom:
```
Comment (Magnus-Kuhn) [inline _docs_operate/setup-with-helm-charts.md:136]:
```suggestion
        valueFrom:
```

File: _docs_operate/setup-with-helm-charts.md [hunk 2]
```diff
@@ -130,6 +132,11 @@ config:
...
+          secretKeyRef:
+            name: base-url
+            key: VALUE
       - name: transportLibrary__platformClientId
         alueFrom:
```
Comment (Magnus-Kuhn) [inline _docs_operate/setup-with-helm-charts.md:141]:
```suggestion
        valueFrom:
```

**Review by Magnus-Kuhn — COMMENTED** [review]
Apart from the typo, the helm charts look right as far as I can tell

File: _docs_operate/setup-with-helm-charts.md (hunk 2 above)
Comment (jkoenig134) [reply _docs_operate/setup-with-helm-charts.md:141]: good catch! I copied from the wrong place!

**Review by Magnus-Kuhn — APPROVED** [review]

**Pull request merged**